src/httpClient/applyDefaultInterceptors.js

```diff
@@ -20,7 +20,7 @@ export default (store, client) => {
         uid,
       };
     }
-    config.data = humps.decamelizeKeys(data);
+    config.data = data ? humps.decamelizeKeys(data) : null;
```

Comment:
FYI: this is for avoid header error in GET request. Without it, axios remove the 'Content-Type' header and the request have a 406 error.